puzzle/blueprints/public/templates/case.html

```diff
@@ -114,6 +114,46 @@
 				</ul>
 			</div>
 		</div>
+
+		<div class="col-md-6">
+			<div class="panel panel-default">
+				<div class="panel-heading">Resources</div>
+
+				<ul class="list-group">
+					{% for resource in case.resources %}
+						<li class="list-group-item">
+							<a href="{{ url_for('public.resource', resource_id=resource.id) }}" title="{{ resource.name }}">{{ resource.name }}</a>
+
+							<form class="badge" action="{{ url_for('public.delete_resource', resource_id=resource.id) }}" method="POST" accept-charset="utf-8">
+									<button class="no-style" type="submit">
+										<span class="glyphicon glyphicon-remove"></span>
+									</button>
+								</form>
+						</li>
+
+					{% endfor %}
+				</ul>
+
+				<div class="panel-body">
+					<form action="{{ url_for('public.resources') }}" method="POST" enctype="multipart/form-data">
+						<div class="form-group">
+							<input class="form-control" name="name" placeholder="resource name"></input>
+						</div>
+						<div class="form-group row">
+							<div class="col-xs-4">
+								<input class="form-control" type="file" name="file"></input>
+							</div>
+							<div class="col-xs-4">
+								{{ utils.select_input('ind_id', case.individuals|map(attribute='ind_id')) }}
+							</div>
+							<div class="col-xs-4">
+								<button class="btn btn-default form-control" type="submit">Upload</button>
+							</div>
+						</div>
+					</form>
+				</div>
+			</div>
+		</div>
 	</div>
 {% endif %}
 
```

Comment:
I'll use 2 spaces for HTML to make it consistent without too much spacing :)
http://stackoverflow.com/a/5936506